.github/dependabot.yml

```diff
@@ -6,5 +6,3 @@ updates:
     interval: daily
     time: "07:00"
   open-pull-requests-limit: 99
-  ignore:
-  - dependency-name: ember-cli-babel
```

Comment:
We want to ignore `ember-cli-babel` because it's a breaking change whenever we update it (so it needs to be released in a major version).

Reply:
Ah, gotcha - Assumed a specific version was broken with the old ember-cli or something.